webapp/channels/src/root.html

```diff
@@ -21,6 +21,9 @@
 
     <!-- CSS Should always go first -->
     <link rel='stylesheet' class='code_theme'>
+
+    <!-- We need this as soon as possible to show the loading screen -->
+    <link id="initialLoadingScreenCSS" rel='stylesheet' href='/static/css/initial_loading_screen.css'>
```

Comment:
Do you know if this is loaded before or after the CSS bundle that's produced by Webpack? I forget if we can control where Webpack adds those links

Reply:
Other CSS links are added at the end of the head tag by the webpack. So we cannot control if we want to add these after webpack adds theirs, it will also be first.